pages/admin/dashboard/modules.tsx

```diff
@@ -58,20 +48,36 @@ const ModulesPage: Page = () => {
                 </Button>
                 <Button variant="link">Published</Button>
             </Flex>
-            <SimpleGrid minChildWidth="20rem" spacing={10}>
-                {modules.map(
-                    ({ _id, title, toolID, lastUpdated, createdBy }) => (
-                        <ModuleCard
-                            key={_id}
-                            moduleId={_id}
-                            title={title}
-                            tool={toolID}
-                            lastUpdated={lastUpdated}
-                            author={createdBy.join(", ")}
-                        />
-                    ),
-                )}
-            </SimpleGrid>
+        </>
+    );
+};
+
+const Modules: React.FC = () => {
```

Comment:
seem to be getting a type error w `Modules`, also `Tools` and `Builder`

Reply:
Has this been cleared up?